### DISCUSSION TITLE
ImportError: DLL load failed while importing _managers

### DISCUSSION THREAD

**Discussion by TonNew5418** [cyberbotics/webots#5019] [Q&A]
R2021a Webots version, when I try to import managers like this
![image](https://user-images.githubusercontent.com/90179098/183244087-80e8d9e6-0b82-4e07-b683-ff629aad3de6.png)
I get an Import Error
![image](https://user-images.githubusercontent.com/90179098/183244101-d5b2b110-fed5-42a9-a015-abbcc8fbacb6.png)
 Could anyone help me out with that, thanks

**Comment by stefaniapedrazzi** [comment]
The import code seems correct.

Could you add the following instruction to your controller and check that the python version is "python37"?
```
import sys
print('python%d%d' % (sys.version_info[0], sys.version_info[1]))
```

Could you also try to run the default sample simulation `samples/robotbenchmark/humanoid_marathon/worlds/humanoid_marathon`?
You can open it from the Webots "File > Open Sample World" menu option.
Is the import working correctly in this simulation?

**Reply by Zyed816** [reply]
I have the same question and the default sample simulation "humanoid_marathon" can be runed correctly .

**Reply by fancybubbles** [reply]
yeah same...

**Comment by AlexYiWang** [comment]
have you fixed the problem?

**Comment by stefaniapedrazzi** [comment]
Additional information are needed to inspect the issue.
